src/components/CourseInfo/CourseInfo.jsx

```diff
@@ -0,0 +1,52 @@
+import { Link, useParams } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faArrowLeft } from '@fortawesome/free-solid-svg-icons';
+import { mockedCoursesList, mockedAuthorsList } from '../../constants';
+import addAuthors from '../../helpers/addAuthors';
+import pipeDuration from '../../helpers/pipeDuration';
+import dateGenerator from '../../helpers/dateGenerator';
+import './courseInfo.css';
+import NotFound from '../NotFound/NotFound';
+
+function CourseInfo() {
+	let { courseID } = useParams();
+	let course = mockedCoursesList.find((item) => item.id === courseID);
```

Comment:
1. It will be beneficial to use `useMemo` here
2. You are trying to find a course in `mockedCoursesList` but what if there were new courses created? it will not be on this list